fern/api/openapi/openapi.yaml

```diff
@@ -4294,30 +4294,31 @@ paths:
       summary: Set revocation registry state manually
       tags:
       - revocation
-  /revocation/registry/{rev_reg_id}/tails-file:
-    get:
-      operationId: download_tails_file
-      parameters:
-      - description: Revocation Registry identifier
-        explode: false
-        in: path
-        name: rev_reg_id
-        required: true
-        schema:
-          pattern: "^([123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{21,22}):4:([123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{21,22}):3:CL:(([1-9][0-9]*)|([123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{21,22}:2:.+:[0-9.]+))(:.+)?:CL_ACCUM:(.+$)"
-          type: string
-        style: simple
-      responses:
-        "200":
-          content:
-            application/octet-stream:
-              schema:
-                format: binary
-                type: string
-          description: tails file
-      summary: Download tails file
-      tags:
-      - revocation
+  # Fern's FastAPI generator doesn't support file download
+  # /revocation/registry/{rev_reg_id}/tails-file:
```

Comment:
This endpoint is commented out because Fern's FastAPI generator does not support generating endpoints with file download. You are welcome to contribute that though!